### ISSUE TITLE
Fortune titles

### ISSUE THREAD

**Issue by CandyAngel** [mojolicious/mojolicious.org#10]
Make the site title (currently "What are you waiting for?") be randomly selected from a "fortune" file (or configuration).

This can contain some plain titles, but also some fun/witty ones. Perhaps also include examples of short bits of code which show off Mojolicious' power (Ojo).

Likely needs to have a character limit when deciding what to add.

```
Bread, Milk, Eggs and Mojo
9 out of 10 penguins recommend Mojolicious!
There's Perl in my framework!
my $choice = c(@frameworks)->grep(qr/Mojolicious/)->first;
n { increase_power() } 9000;
j($ust).. do it!
```

**Comment by marcusramberg** [comment]
```
Mojolicious - It's the bee's knees.
Semper Mojo
Cognito Ergo Mojolicious
Mojo::IOLoop - There can only be one!
Gimme a M, gimme a O, gimme a J, gimme an O
One does not simply write a web app without Mojolicious.
Brace Yourself, Mojolicious is coming.
I am Mojo
It's as if a million lines of cgi cried out and were suddenly silenced.
Yo, Imma let you finish, but Mojolicious is the best web framework of all time!
```

**Comment by dotandimet** [comment]
Mojo doesn't socialize

**Comment by ilmari** [comment]
```
It's not just delicious, it's Mojolicious!
```

**Comment by preaction** [comment]
`Fun is job 1` (from this old Ford commercial slogan "Quality is job 1": https://www.youtube.com/watch?v=X3ecGr_-2vQ)
`Do not taunt happy fun web framework` (from old SNL skit: https://en.wikipedia.org/wiki/Happy_Fun_Ball)
`Not legal tender` (or just all of the Futurama title captions: https://theinfosphere.org/List_of_title_captions, specifically "Hey TIOBE! Suggest this!")

**Comment by dotandimet** [comment]
Every deploy, a Moment of Joy.

**Comment by CandyAngel** [comment]
Implemented in #11.
